packages/apps-config/src/links/subscan.ts

```diff
@@ -13,7 +13,8 @@ export default {
     'Kusama CC3': 'kusama',
     Polkadot: 'polkadot',
     'Polkadot CC1': 'polkadot-cc1',
-    Westend: 'westend'
+    Westend: 'westend',
+    'Centrifuge Mainnet': 'centrifuge'
```

Comment:
Just move this up to first position (Alphabetical)

Reply:
yeap :-D